plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/messages.properties

```diff
@@ -2,4 +2,5 @@ application.element=Project ID should be specified at deploy time
 remove.application.element=Remove application element
 web.xml.version=App Engine Standard does not support this servlet version
 convert.servlet=Convert to Java Servlet API 2.5
-maven.plugin=This App Engine Maven plugin has been deprecated
\ No newline at end of file
+maven.plugin=This App Engine Maven plugin has been deprecated
+undefined.servlet.class={0} does not exist in the project
```

Comment:
The class could be pulled from a library.  `{0} could not be found`?  `could not be resolved`?